src/commonTest/kotlin/fr/acinq/lightning/channel/states/SpliceTestsCommon.kt

```diff
@@ -22,20 +26,33 @@ import fr.acinq.lightning.wire.*
 import kotlinx.coroutines.CompletableDeferred
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.runBlocking
+import kotlin.math.abs
 import kotlin.test.*
 
 class SpliceTestsCommon : LightningTestSuite() {
 
     @Test
     fun `splice funds out`() {
         val (alice, bob) = reachNormal()
-        spliceOut(alice, bob, 50_000.sat)
+        val (nodes, preimage, _) = addHtlc(15_000_000.msat, alice, bob)
+        val (alice1, bob1) = crossSign(nodes.first, nodes.second)
+        val (alice2, bob2) = spliceOut(alice1, bob1, 50_000.sat)
+        val fee = spliceFee(alice2, capacity = 950_000.sat)
+        fulfillHtlc(0, preimage, alice2, bob2)
+        assertEquals(alice2.state.commitments.latest.localCommit.spec.toLocal, 750_000_000.msat - fee.toMilliSatoshi() - 15_000_000.msat)
+        assertEquals(alice2.state.commitments.latest.localCommit.spec.toRemote, 200_000_000.msat)
```

Comment:
Changes to this test and the following one are redundant with the other tests that contain HTLCs, we could keep those unchanged to have tests for the case where no HTLCs are in the commitment?